### ISSUE TITLE
Pagination source not displayed

### ISSUE THREAD

**Issue by jonaslil** [w3c/publ-a11y#484]
EPUB Accessibility 1.1 states:

> Users need to know the source of the pagination in an EPUB publication to determine whether it will be useful for their needs.
> ...
> When an EPUB publication includes page break markers and/or a page list that correspond to a statically-paginated version of the publication, EPUB creators MUST identify that source in the package document metadata.

For this, the `a11y:pageBreakSource` property exists.

Recently, support for identifying the pagination source for reflowable EPUBs using this property was added to InDesign.

ONIX provides a related code (13 in Codelist 51) to identify that a digital product corresponds to a specific print edition. 

However, neither the Display Guide nor the Techniques mention displaying the pagination source to users.

This raises a question: Why are content creators required to include this metadata, and why do we promote adding support for it in production software, if it is not displayed to users?

**Comment by gautierchomel** [comment]
I agree with this point; it's important, but "displaying" it may not be the best approach. 

Some considerations that I guess lead us not to integrate it before:
* The source is preferably a unique identifier, making it poorly understandable for non-professionals. As a reader, Am I supposed to copy and paste that unique identifier to find the source? 
* Early French recommendations had it, but it was not implemented, and booksellers (service providers of bookselling platforms) explained that they use another mechanism to link different versions of a book (prints, audio, digital) so the reader can identify different formats on the same page. 
* Some collections we have seen (including Daisy's specialised catalogues) have "print pages" and "source" but no correspondence between both, making the information false. 
* Not all metadata are meant to be displayed; some are of better use in BtoB systems

That said, we should include a note or indication that the print pagination source should be identifiable (linked or displayed, depending on the audience).

**Comment by jonaslil** [comment]
Thanks Gautier for the explanation. I agree that displaying a unique identifier, such as an ISBN, isn't very user friendly, and adding a note about this is a great idea.

**Comment by GeorgeKerscher** [comment]
This is an interesting problem. I agree that many people will want to know the print source. Some ideas:

The reading system could present this information to the user, but this would be after the publication was purchased/acquired, which is not great.

The metadata could be presented, but on its own is not too useful, unless it was resolved to the medata of the source, which also could add to confusion.

The print source could be to a paperback or to a hard cover version which would probably have different pages; are the identifiers different?

Finally there is the accessibility summary which could be used to provide specifics.  For example: the publisher could in the accessibility summary that the print source for the publication is:

Then give the details.

The question remains for us on what to do in the guidelines.

**Comment by HadrienGardeur** [comment]
Distributors often send relationships between editions through ONIX, for example:

- a link back to the print edition that an ebook version is based on
- or a link to an audiobook edition

Retailers and libraries can use these references in their catalogs in order to group editions together or provide links between them.

In theory, this EPUB metadata would allow a completely different reference (to another print edition for example). In practice, I'm not sure that anyone will be able to use this EPUB metadata.

The best case scenario that I can imagine:

- some retailers/libraries have access to files in addition to ONIX
- when they detect relationships between editions, they could also check for the presence of this metadata in EPUB
- based on these information, they could further contextualize what they display to the user by either:
  - providing a link to the print edition
  - and/or adding further metadata such as the total number of pages from the print edition

From a reading system perspective, I don't think that they'll be able to do anything at all. Reading systems need to interact with a catalog (in OPDS for example) to do anything useful with this metadata.

**Comment by gautierchomel** [comment]
FYI, we opened a discussion to provide a means to identify print page sources in Thorium Reader: [Thorium provides no mean to identify print page source #2726](https://github.com/edrlab/thorium-reader/issues/2726)

**Comment by GeorgeKerscher** [comment]
Thorium would like to keep this open to see if they could display something

**Comment by GeorgeKerscher** [comment]
Thorium has their own issue, so closing this.
